.eslintrc.json

```diff
@@ -0,0 +1,30 @@
+{
+  "env": {
+    "browser": true,
+    "jasmine": true,
+    "jquery": true,
+    "amd": true
+  },
+  "extends": "airbnb-base/legacy",
+  "globals": {
```

Comment:
Ah, very cool. I didn't realize that the env statements also imply additional rules.